pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -924,16 +924,17 @@ func applyToTargetCluster(
 		return err
 	}
 
+	logger.WithField("uid", obj.GetUID()).Info("attempting to apply resource")
 	applyResult, err := applyFn(bytes)
 	// Record the amount of time we took to apply this specific resource. When combined with the metric for duration of
 	// our kube client requests, we can get an idea how much time we're spending cpu bound vs network bound.
 	applyTime := metav1.Now().Sub(startTime).Seconds()
 	if err != nil {
-		logger.WithError(err).Warn("error applying resource")
+		logger.WithError(err).WithField("uid", obj.GetUID()).WithField("applyTime", applyTime).Warn("error applying resource")
```

Comment:
✓ I will be interested to see what the per-resource values are for `applyTime` in stage.